src/core/github.js

```diff
@@ -9,6 +9,44 @@ import { pub } from "core/pubsubhub";
 
 export const name = "core/github";
 
+var body = document.body;
+var head = document.head;
+var issueBase;
+
+var fileIssueLink = document.createElement('button');
+fileIssueLink.setAttribute('data-balloon','file an issue related to the text selected');
+fileIssueLink.setAttribute('data-balloon-pos','left');
+fileIssueLink.classList.add('removeOnSave');
+fileIssueLink.id = 'respec-file-issue-btn';
+fileIssueLink.innerHTML = 'File Issue';
+fileIssueLink.onclick = function () {
+  window.location.href = fileIssueLink.getAttribute('data-href');
+  fileIssueLink.style.display = 'none';
+};
+
+var link = document.createElement('link');
+link.rel = 'stylesheet';
+link.href = "https://cdnjs.cloudflare.com/ajax/libs/balloon-css/0.5.0/balloon.min.css";
+
+head.appendChild(link);
+body.appendChild(fileIssueLink);
+
+body.onmouseup = function () {
+  var textSelected = window.getSelection().toString();
```

Comment:
What happens if the use selects the whole document? We should limit the length of the selection to something reasonable.

Reply:
Hmm, any suggestions to what the length should be?

Reply:
For the title, I would say, maybe 40 characters and then from there find the last whitespace character (so we don't chop words in half)... if the length was exceed, we should add "..." on the end. 

For the body, it can be much larger... but we need to test that out, to make sure we format it properly in the bug (e.g., it should appear as quoted text on github... we can get fancier from there, but this would be a great start).

Reply:
Cool. I will incorporate these changes too. :+1: